StoreApp.BLL/Services/UserValidateService.cs

```diff
@@ -41,5 +45,20 @@ public string GetRole(UserDTO userDTO)
 
             return users.FirstOrDefault(u => u.UserName.Equals(userDTO.UserName) && u.Password.Equals(userDTO.Password))?.Role;
         }
+
+        public bool CheckTruePassword(UserDTO userDTO)
+        {
+            return DataBase.Users.GetAll()
+                                 .Any(u => u.Id.Equals(userDTO.Id) &&
+                                           u.Password.Equals(userDTO.Password));
+        }
+
+        public bool CheckForEditUser(UserDTO userDTO)
+        {
+            return !DataBase.Users.GetAll()
+                                  .Where(u => u.Id != userDTO.Id)
+                                  .Any(u => u.UserName.Equals(userDTO.UserName) ||
+                                            u.Email.Equals(userDTO.Email));
```

Comment:
1. Just apply default formatting. 
2. Why have you applied such an approach? It seems to me you can use only **Any**

Reply:
fixed